### PR TITLE
Add dexOrderDaoTest, add async cancel order processing

### PR DESCRIPTION
**Pull request by AndrewBoyarsky**

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
-import com.apollocurrency.aplwallet.apl.exchange.model.DexContractDBRequest;
-import com.apollocurrency.aplwallet.apl.exchange.model.DexCurrencies;
-import com.apollocurrency.aplwallet.apl.exchange.model.DexOrder;
-import com.apollocurrency.aplwallet.apl.exchange.model.DexOrderDBRequest;
-import com.apollocurrency.aplwallet.apl.exchange.model.ExchangeContract;
-import com.apollocurrency.aplwallet.apl.exchange.model.ExchangeContractStatus;
-import com.apollocurrency.aplwallet.apl.exchange.model.OrderStatus;
-import com.apollocurrency.aplwallet.apl.exchange.model.OrderType;
-import com.apollocurrency.aplwallet.apl.exchange.model.TransferTransactionInfo;
+import com.apollocurrency.aplwallet.apl.exchange.model.*;
```
Comment (nemezxxx) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java]: that kind of import is unacceptable; 
import only these things which are really necessary to you and nothing apart from that

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java [hunk 2]
```diff
@@ -41,11 +35,13 @@
 @Slf4j
 @Singleton
 public class DexOrderProcessor {
+    private static final int ORDERS_SELECT_SIZE = 30;
```
Comment (nemezxxx) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java]: how about moving these things to Constants?

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java (hunk 1 above)
Comment (AndrewBoyarsky) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java]: Fixed

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java (hunk 2 above)
Comment (AndrewBoyarsky) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/service/DexOrderProcessor.java]: Is not necessary, used only for one method here

File: apl-core/src/test/java/com/apollocurrency/aplwallet/apl/exchange/dao/DexOrderDaoTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import org.jboss.weld.junit5.EnableWeld;
+import org.jboss.weld.junit5.WeldInitiator;
+import org.jboss.weld.junit5.WeldSetup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (nemezxxx) [inline apl-core/src/test/java/com/apollocurrency/aplwallet/apl/exchange/dao/DexOrderDaoTest.java]: Again. Import ONLY what you need and nothing apart from that

**Pull request merged**